CodeAssignment_MVVM_Combine/Sources/Controllers/DetailViewController.swift

```diff
@@ -0,0 +1,309 @@
+        label.setContentHuggingPriority(.init(999), for: .horizontal)
+        label.setContentCompressionResistancePriority(.required, for: .horizontal)
+        return label
+    }()
+    
+    private lazy var watchersCountLabel: UILabel = makeCountLabel()
+    private lazy var forksCountLabel: UILabel = makeCountLabel()
+    private lazy var openIssuesCountLabel: UILabel = makeCountLabel()
+    /// starButton, languageColorView, languageNameLabelを持つStackView
+    private lazy var starLanguageStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [starButton, languageColorView, languageNameLabel])
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.spacing = 8
+        return stackView
+    }()
+    /** starCountLabelを除いたCountLabelを持つStackView
+    - constraintを一概に設定しやすいし、layout設定に関するコードが長くなることを防ぐ
+     */
+    private lazy var countStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [watchersCountLabel, forksCountLabel, openIssuesCountLabel])
+        stackView.axis = .vertical
+        // subView間のspacing設定
+        stackView.spacing = 12
+        stackView.alignment = .trailing
+        return stackView
+    }()
+    /// backgroundViewに入れるSubViewを持つStackView
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [
+            userImageContentView, userNameLabel, repositoryNameLabel, descriptionLabel, starLanguageStackView, countStackView
+        ])
+        stackView.axis = .vertical
+        stackView.spacing = 20
+        return stackView
+    }()
+    
+    init(repository: Repositories.Repository) {
+        super.init(nibName: nil, bundle: nil)
+        self.configure(with: repository)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+}
+
+extension DetailViewController {
+    private func configure(with model: Repositories.Repository) {
+        let defaultImage = UIImage(systemName: "person.circle")?.withTintColor(.systemMint, renderingMode: .alwaysOriginal)
+        
+        repositoryNameLabel.text = model.name
+        descriptionLabel.text = model.description
+        userNameLabel.text = model.owner.userName
+        languageNameLabel.text = model.language
+        // UIButton.Configurationのtitleを更新
+        if var config = starButton.configuration {
+            config.attributedTitle = AttributedString(
+                "\(formatNumberToStringWithSeparator(model.stargazersCount)) stars",
+                attributes: AttributeContainer([
+                    .font: UIFont.systemFont(ofSize: 20, weight: .regular),
+                            .foregroundColor: UIColor.black.withAlphaComponent(0.8)
+                ]))
+            starButton.configuration = config
+        }
```

Comment:
ここは if varでやってもいいけど、↓とやってること変わらないからこっちのほうがシンプルでいいかも。
わざわざconfigurationを入れ直してるのも冗長かなと思うので
```suggestion
        starButton.configuration?.attributedTitle = AttributedString(
            "\(formatNumberToStringWithSeparator(model.stargazersCount)) stars",
            attributes: AttributeContainer([
                .font: UIFont.systemFont(ofSize: 20, weight: .regular),
                .foregroundColor: UIColor.black.withAlphaComponent(0.8)
            ])
        )
```

Reply:
[d86806d]にてコード修正しました！ご確認お願いします！